orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/utils/OrientDBTest.java

```diff
@@ -6,11 +6,37 @@
  */
 package org.hibernate.datastore.ogm.orientdb.utils;
 
+import com.tinkerpop.blueprints.impls.orient.OrientGraph;
 import org.hibernate.ogm.backendtck.simpleentity.CRUDTest;
+import org.junit.Before;
 
 /**
  * @author Davide D'Alto
  */
 public class OrientDBTest extends CRUDTest {
 
+	private static final String TEST_DB_URL = "memory:/tmp/orientdb_ogm_test";
+
+	/**
+	 * Drop and create the test classes (tables) schema in OrientDB
+	 */
+	@Before
+	public void intialize() {
```

Comment:
Ah I see now, ideally we should do this when the session factory is created.
Using the `SchemaDefiner` I was talking about before